build.sh

```diff
@@ -62,6 +71,20 @@ case "${BRANCH}" in
 esac
 DOCKER_TAG="${DOCKER_TAG-${DOCKER_ORG}/${DOCKER_REPO}:${TAG}}"
 
+# Checking which VARIANT to build
+if [ -z "$VARIANT" ]
```

Comment:
Just for consistency with the rest of the code, could you please write the `if` on one line:

```bash
if true; then
  echo ✨
fi
```